### ISSUE TITLE
sudo katoolin

### ISSUE THREAD

**Issue by OniyaGriseo1122** [LionSec/katoolin#243]
I get this response when executing the final command : "sudo: unable to execute /usr/bin/katoolin: No such file or directory". Thanks for the great resource and the help.

**Comment by SimplyPancake** [comment]
Try
`sudo chmod +x /usr/bin/katoolin`

**Comment by keithsavage777** [comment]
i did and still get the same as Oniya boss.sudo: unable to execute /usr/bin/katoolin: No such file or directory". Thanks for the great resource and the help. the idea was to install kita tools on my ubuntu 20-04 and hum. maybe a code error in katoolin.py or my py not updated?

**Comment by mendyz** [comment]
same issue here in wsl 2 ubuntu 20.04

**Comment by RiceGod20** [comment]
I have the same problem. I do all the steps correctly but when I put in sudo katoolin it does not work. I am on a chromebook.

**Comment by freedom-wy** [comment]
i have same problem.ubuntu18.04
```
sudo katoolin 
sudo: unable to execute /usr/bin/katoolin: No such file or directory
```

**Comment by freedom-wy** [comment]
solve the problem
`sudo python2.7 katoolin.py`

**Comment by OniyaGriseo1122** [comment]
Ah! Glad to see some others found this thread beneficial; thanks, freedom-wy, for the fix and what I'm sure is a magnificent tool when it's working. Many blessings.

**Comment by OniyaGriseo1122** [comment]
Yes sir, that one did it. I missed this being Python, sudo often doesn't work with python stuff. Use pip or conda but not both, mixing installers leads to a destabilized system. Also you've got to invoke Python before you run a program. Most of you know but for those who might not, there you are.

**Comment by LagMan932** [comment]
Hi, I have the same issue: chmod +x command doesn't work and even trying "sudo python2.7 katoolin.py" I can't use katoolin. Can someone help me?

**Comment by defaultRectr0** [comment]
Try this command : python2 /usr/bin/katoolin

**Comment by Revitte** [comment]
It worked! but needs "sudo" privileges
